src/Turbo/Bridge/Mercure/Broadcaster.php

```diff
@@ -71,6 +72,15 @@ public function broadcast(object $entity, string $action, array $options): void
 
         $options['topics'] = (array) ($options['topics'] ?? sprintf(self::TOPIC_PATTERN, rawurlencode($entityClass), rawurlencode(implode('-', (array) $options['id']))));
 
+        if (isset($options['scope'])) {
+            $expressionLanguage = new ExpressionLanguage();
+            $scope = $expressionLanguage->evaluate($options['scope'], [
+                'entity' => $entity,
+            ]);
+
+            $options['topics'][] = sprintf(self::TOPIC_PATTERN, rawurlencode($entityClass.'_'.$scope), rawurlencode(implode('-', (array) $options['id'])));
```

Comment:
Question about this. If I'm passing in a "scope", do I want the update to be published BOTH to the "scoped topic" AND also the normal topic from above? Or ONLY the scoped topic? It looks like you purposely published to both topics - I'd like to know your thinking on that.

Reply:
Let's take the Book and Author example again, just like the tests.
Imagine you want to show a list of every newly created book. By publishing to both topics we retain this possibility. You can subscribe to the scoped topic for books for a specific author, and the non-scoped topic for any book.

Reply:
This discussion no longer relevant with the new approach